src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/MongoConfig.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.overseasentitiesapi.configuration;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.convert.MappingMongoConverter;
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+import static uk.gov.companieshouse.overseasentitiesapi.configuration.MongoConverters.getMongoCustomConversions;
+
+@Configuration
+@EnableMongoRepositories(basePackages = "uk.gov.companieshouse.overseasentitiesapi.repository")
+public class MongoConfig {
+
+    private static final String MONGO_DB_URI_PROPERTY = "spring.data.mongodb.uri";
+    private static final String MONGO_DB_NAME_PROPERTY = "overseas.entities.mongodb.dbname";
+
+    @Bean
+    public MongoClient mongoClient(@NotNull Environment environment) {
+        final ConnectionString connectionString = new ConnectionString(
```

Comment:
How about adding a log line in each of these methods, just so we know that the new config is actively being used?